docker-compose/README.md

```diff
@@ -74,30 +71,33 @@ After executing the script, the needed set of data directories for Artifactory P
 
 ---
 ### Database Driver
-The database used in these examples is PostgreSQL.  
+The database used in these examples is PostgreSQL 9.5.2.  
```

Comment:
No need for the actual version as this might change will keep this README wrong.